src/django/api/management/commands/replace_oar_id.py

```diff
@@ -0,0 +1,61 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from api.models import FacilityListItem, FacilityMatch
+
+
+class Command(BaseCommand):
+    help = ('Replace references to oar_id in item processing_results and '
+            'match results')
+
+    def add_arguments(self, parser):
+        parser.add_argument('-d', '--dry-run', action='store_true',
+                            default=False, help='Do not save cahnges')
+
+    def handle(self, *args, **options):
+        dry_run = options['dry_run']
+
+        if dry_run:
+            print('== DRY RUN - SKIPPING ALL SAVES')
+
+            print('-- PROCESSING ITEMS')
```

Comment:
Suggest decreasing indent so this print outside of dry runs

Reply:
Good catch. Thank you.

Reply:
Fixed in the rebase.